### ISSUE TITLE
Sorting does not work on multipages

### ISSUE THREAD

**Issue by Falco20019** [Analogy-LogViewer/Analogy.LogViewer.Serilog#715]
I have 250.000 messages. The LogViewer splits them onto 2 pages. When sorting by date, I would assume that they are FIRST sorted and THEN paginated. But page 1 even has data that is AFTER page 2's earliest dates. So it seems that the data is first split into pages and then sorted per page (which doesn't make sense).

**Comment by LiorBanai** [comment]
for now you can increase page size but this will impact performance. reorder pages on each sort may also impact performance... I'll check both options

**Comment by Falco20019** [comment]
Yep, I disabled paging (highly decreasing usability) to circumvent the problem. When you know which files the logs you search are in, only loading those also helped.

**Comment by LiorBanai** [comment]
I'll check. Maybe resorting all pages won't be heavy task.
